client/state/help/directly/reducer.js

```diff
@@ -9,10 +9,8 @@ import { STATUS_UNINITIALIZED, STATUS_INITIALIZING, STATUS_READY, STATUS_ERROR }
 
 export const questionAsked = ( state = null, action ) => {
 	switch ( action.type ) {
-		case DIRECTLY_ASK_QUESTION: {
-			const { questionText, name, email } = action;
-			return { questionText, name, email };
-		}
+		case DIRECTLY_ASK_QUESTION:
+			return true;
```

Comment:
This change makes sense, we don't care about the question details, but only whether it's been asked or not.